Multy/BricksView.swift

```diff
@@ -0,0 +1,57 @@
+//Copyright 2017 Idealnaya rabota LLC
+//Licensed under Multy.io license.
+//See LICENSE for details
+
+import UIKit
+
+class BricksView: UIView {
+    var currentCheckedWordCounter : Int = 3;
+    var segmentsCountUp : Int  = 7
+    var segmentsCountDown : Int  = 8
+    
+    let upperSizes : [CGFloat] = [0, 35, 79, 107, 151, 183, 218, 253]
+    let downSizes : [CGFloat] = [0, 23, 40, 53, 81, 136, 153, 197, 249]
+    
+    init(with rect : CGRect, and currentWordCounter: Int) {
+        super.init(frame: rect)
+        backgroundColor = UIColor(red: 249.0/255.0, green: 250.0/255.0, blue: 1.0, alpha: 1.0)
+        currentCheckedWordCounter = currentWordCounter
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    // Only override draw() if you perform custom drawing.
+    // An empty implementation adversely affects performance during animation.
+    override func draw(_ rect: CGRect) {
+        
```

Comment:
It is not obvious what you do here, please add a function-level comment.

Reply:
Add new comment in code